cmd/testrunner/main.go

```diff
@@ -174,11 +178,17 @@ func deployOperator(f flags, c client.Client) error {
 		return errors.Errorf("error building operator role binding: %s", err)
 	}
 	fmt.Println("Successfully created the operator Role Binding")
+	watchNsp := f.namespace
```

Comment:
Same here about the variable name? `watchNamespace` sounds better to me, even though `watchNsp` is absolutely fine to understand

Reply:
Sounds good! I made the change.